### ISSUE TITLE
Fix function names in nfCommon

### ISSUE THREAD

**Issue by lookeypl** [nfprojects/nfengine#107]
Some modules in nfCommon still use old function names, beginning with X, to name its internal functions, or APIs.
- Unify these names so that they will fit rest of the project
- Check if other modules have unified names as well. Especially make sure, that API names are correct.

Modules containing such functions:
- Image
- Math/Vector

**Comment by lookeypl** [comment]
I found more functions with X at the beginning, in Math/Vector.hpp. Issue updated.

**Comment by mkulagowski** [comment]
You can assign me. 
I've run through all files in nfCommon folder and found following functions:

Image.cpp: 
    XImage_GetTexel
    XImage_SetTexel
    XImage_DDSGetFormat

Math/Vector.hpp:
    XPlaneFromPoints
    XPlanePointDot3

These are only functions in files from nfCommon folder with names beggining with 'X'.

**Comment by lookeypl** [comment]
Okay,  I did some thinking about it and here is what follows:

As for Math/Vector.hpp it is a simple job - these are wrapped in NFE::Math namespace, so just remove the 'X' at the beginning and we should be fine. Of course, check where these functions are called and correct it there as well.

As for Image.cpp, I'm thinking that nowadays these functions do not need __forceinline (compiler will decided it during optimization process for himself). So, since these functions are a part of internal Image.cpp functions, remove __forceinline and check if you can make them a private member of Image class. However, keep in mind, that:
1. This might require some changes to these functions. @Witek902 will know more about it, since he is the author of this code.
2. Image Module still does not have its tests in nfCommonTest project yet, so any huge changes might destroy something we don't know about.

Long story short - change the names, if converting Get/SetTexel and DDSGetFormat to private members requires more work than just moving these functions, it is safer to leave it for a separate task.

**Comment by mkulagowski** [comment]
https://review.gerrithub.io/#/c/217757/

**Comment by lookeypl** [comment]
Fix merged, closing.
